### PR TITLE
feat: code quality

### PR DESCRIPTION
**Pull request by dni**

### CODE REVIEW

File: crud.py [hunk 1]
```diff
@@ -125,34 +131,30 @@ async def delete_charge(charge_id: str) -> None:
...
+                    pending=unconfirmed,
+                )
+        except Exception as e:
+            logger.warning(e)
+    if charge.lnbitswallet and charge.payment_hash:
+        payment = await get_standalone_payment(charge.payment_hash)
+        status = (await payment.check_status()).success if payment else False
+        if status:
+            return await update_charge(charge_id=charge_id, balance=charge.amount)
+    return None
```
Comment (motorina0) [inline crud.py]: the old code was always returning an instance no matter of the if-else branches and of the errors:
```py
return await get_charge(charge_id)
```

File: models.py [hunk 2]
```diff
@@ -62,9 +61,10 @@ def from_row(cls, row: Row) -> "Charges":
 
     @property
     def time_left(self):
+        assert self.last_accessed_at, "Charge has not been accessed yet."
```
Comment (motorina0) [inline models.py:64]: why is this required?

Comment (dni) [reply models.py:64]: its optional... on models so the calculation wouldnt work is none

File: crud.py (hunk 1 above)
Comment (dni) [reply crud.py]: done

File: crud.py [hunk 3]
```diff
@@ -125,33 +131,29 @@ async def delete_charge(charge_id: str) -> None:
...
-
-                    await update_charge(
-                        charge_id=charge_id,
-                        balance=confirmed,
-                        pending=unconfirmed,
-                    )
-            except Exception as e:
-                logger.warning(e)
-        if charge.lnbitswallet:
-            try:
```
Comment (motorina0) [inline crud.py:146]: why remove the try-catch?

Comment (dni) [reply crud.py:146]: nothing can fail here
